src/components/shop-list.js

```diff
@@ -70,30 +63,25 @@ class ShopList extends Element {
     </style>
 
     <shop-image
-        alt="[[category.title]]"
-        src="[[category.image]]"
-        placeholder-img="[[category.placeholder]]" class="hero-image"></shop-image>
+        alt="${category.title}"
```

Comment:
how come these don't have to be `$=`?

Reply:
`alt` is reflected to attribute by `<img>`

Reply:
But below you had `<a href$="`, maybe that should be `=` instead, then?

Reply:
(like both of these are properties and attributes, no?)

Reply:
Ah yes, will do. With PolymerElement `href$=[[falsyValue]]` would remove the attribute, but we don't have that with LitElement so we can't rely on that